pkg/api/routes_studio.go

```diff
@@ -47,7 +49,12 @@ func (rs studioRoutes) Image(w http.ResponseWriter, r *http.Request) {
 	}
 
 	if err := utils.ServeImage(image, w, r); err != nil {
-		logger.Warnf("error serving studio image: %v", err)
+		// Broken pipe errors are common when serving images and the remote
+		// connection closes the connection. Filter them out of the error
+		// messages, as they are benign.
+		if errors.Is(err, syscall.EPIPE) {
```

Comment:
Shouldn't this be the inverse?

Reply:
Yeah, it is late :) Fixed in the next push here.